Stardaze/OutputFormatter.swift

```diff
@@ -17,6 +23,160 @@ internal struct PrettyPrintedStringFormatter: Visitor {
         return finishedString
     }
 
+    private func makeCondensedQuery(document: Document) -> NSMutableString {
+        var query =  visit(document.queryOperation)
+
+        if let fragments = document.fragments {
+            let fragments = visit(fragments)
+
+            query.append(" ")
+            query.append(fragments)
+        }
+
+        let condensedQuery = NSMutableString(string: query)
+        condensedQuery.condenseWhitespace()
```

Comment:
All references to NSString and its mutable variant are being deleted as a part of my update to this branch.